rclpy/actions/minimal_action_server/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>examples_rclpy_minimal_action_server</name>
+  <version>0.6.1</version>
+  <description>Examples of minimal action servers using rclpy.</description>
+
+  <maintainer email="sloretz@openrobotics.org">Shane Loretz</maintainer>
```

Comment:
Keep me as the maintainer, but add yourself as `<author>`

Reply:
Added dfd71ad